test_fixtures/vilbert/experiment_detectron.json

```diff
@@ -0,0 +1,60 @@
+{
+  "dataset_reader": {
+    "type": "nlvr2",
+    "image_dir": "/Users/dirkg/Documents/data/vision/nlvr2/images",
```

Comment:
Is it possible to add those images to the github? and relative locations?

Reply:
The way I have it right now, that's 11GB of images. I'll add a subset to GitHub so we can run the test.